Dockerfile

```diff
@@ -0,0 +1,20 @@
+# Use an official Node.js runtime as a parent image
+FROM node:22
+
+# Set the working directory in the container
+WORKDIR /usr/src/app
+
+# Create the cache directory
+RUN mkdir -p ./cache
+
+# Copy the current directory contents into the container at /usr/src/app
+COPY . .
+
+# Install any needed packages specified in package.json
+RUN npm install
+RUN npm update
+# Define environment variable
+ENV NODE_ENV production
+
+# Run the app when the container launches
+ENTRYPOINT ["tail", "-f", "/dev/null"]
```

Comment:
I don't think this is working for me.  When I run the container, it immediately exits.

```
$ docker run -itd actual-helper --name actual-helper
74214d5b863c9ed48dbcb62c700ef7317b0e4e40af51a16349f6a50ad923a187
$ docker ps -a
CONTAINER ID   IMAGE           COMMAND                  CREATED         STATUS                     PORTS     NAMES
74214d5b863c   actual-helper   "tail -f /dev/null -…"   2 seconds ago   Exited (1) 2 seconds ago             determined_turing
```

Reply:
Whoops my bad, the --name needs to go first, it should be like this.

`docker run -itd --name actual-helper actual-helper `

Reply:
Ya that will fix it. Please update the README instructions.